lz4/frame/__init__.py

```diff
@@ -262,7 +263,7 @@ def compress(self, data):  # noqa: F811
 
         return result
 
-    def flush(self):
+    def flush(self) -> Union[bytes, bytearray]:
```

Comment:
*I think* that with some shenanigans with `.pyi` bindings, we could make this concrete based on the boolean that gets passed to the class. I can take a look in another PR

Reply:
You can't, resolving